src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -413,36 +467,16 @@ public static void sendImage(String fileUrl, String fileName, User user) {
         }
     }
 
-    public static boolean canChangeNick(String guildId) {
-        return Getters.getGuildById(guildId) != null &&
-                (Getters.getGuildById(guildId).getSelfMember().hasPermission(Permission.NICKNAME_CHANGE) ||
-                        Getters.getGuildById(guildId).getSelfMember().hasPermission(Permission.NICKNAME_MANAGE));
-    }
-
-    public static String getStackTrace(Throwable e) {
-        StringWriter writer = new StringWriter();
-        PrintWriter printWriter = new PrintWriter(writer);
-        e.printStackTrace(printWriter);
-        printWriter.close();
-        return writer.toString();
-    }
-
-    public static int getInt(String s, int defaultValue) {
-        try {
-            return Integer.parseInt(s);
-        } catch (NumberFormatException e) {
-            return defaultValue;
-        }
-    }
-
-    public static long getLong(String s, long defaultValue) {
-        try {
-            return Long.parseLong(s);
-        } catch (NumberFormatException e) {
-            return defaultValue;
-        }
-    }
-
+    /**
+     * An error handler
+     * (Someone please help me with this ones doc.)
```

Comment:
I'll let you off this one but get it done for the next review ;) <3

Reply:
I have no clue what this is doing so I don't think i should be the one to write this.

Reply:
"Runs code and catches any errors, printing success, error and start messages as appropriate to the specified logger"